packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -0,0 +1,455 @@
+import React from 'react';
+import { localize, Localize } from '@deriv/translations';
+import { Formik, Field } from 'formik';
+import { Checkbox, Button, Input, FormSubmitButton, Modal, Icon, Money, Loading } from '@deriv/components';
+import CurrencyUtils from '@deriv/shared/utils/currency';
+import { connect } from 'Stores/connect';
+import { WS } from 'Services/ws-methods';
+
+const initial_form = {
+    otherFinancialPriorities: false,
+    stopTrading: false,
+    notIntrested: false,
+    anotherTradingWebsite: false,
+    notUserFriendly: false,
+    isDifficult: false,
+    lackFeaturesFunctionality: false,
+    unsatisfactoryCs: false,
+    otherReasons: false,
+    otherTradingPlatforms: '',
+    doToImprove: '',
+};
+
+const preparingReason = (values) => {
+    const selected_reasons = selectedReasons(values)
+        .map((val) => val[0])
+        .toString();
+    return `${selected_reasons}, ${values.otherTradingPlatforms}, ${values.doToImprove}`;
+};
+
+const selectedReasons = (values) => {
+    return Object.entries(values).filter(
+        ([key, value]) => !['otherTradingPlatforms', 'doToImprove'].includes(key) && value
+    );
+};
+
+const validateFields = (values) => {
+    const error = {};
+    const selected_reason_count = selectedReasons(values).length;
+    if (selected_reason_count === 0 && values.otherTradingPlatforms.length === 0 && values.doToImprove.length === 0) {
```

Comment:
```suggestion
    if (!selected_reason_count && !values.otherTradingPlatforms.length && !values.doToImprove.length) {
```

Reply:
this should be exact 0. it's not about checking the existence of the value.
if I change it'll break